sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -474,6 +489,15 @@ export interface TopicRuntimeProperties {
 export interface TopicRuntimePropertiesResponse extends TopicRuntimeProperties, Response {
 }
 
+// @public
+export type UpdatableFieldsForQueue = "defaultMessageTtl" | "lockDuration" | "deadLetteringOnMessageExpiration" | "duplicateDetectionHistoryTimeWindow" | "maxDeliveryCount";
```

Comment:
If we are anyway enumerating the updatable fields and the allow undefined fields, what are we gaining from not simply defining all the properties in `QueueProperties` inline?

There are 17 properties on `CreateQueueOptions`.
3 of them can be undefined in `QueueProperties`
5 of them are updatable in `QueueProperties`
So, the gain is that we don't have to list the rest of the 9 properties inline
This is in favor of creating 2 new types.

Trying to weigh the pros and cons of inlining vs the overhead of adding new types, plus any documentation challenges this brings.

cc @bterlson, @xirzec, @chradek

Reply:
Going with duplicating the fields for CreateEntityOptions and EntityProperties since the docs are bad as per https://github.com/Azure/azure-sdk-for-js/pull/10049#discussion_r458626602